optimade/models/structures.py

```diff
@@ -66,7 +66,7 @@ class Species(BaseModel):
 
     name: str = Field(
         ...,
-        decsription="""Gives the name of the species; the **name** value MUST be unique in the `species` list.""",
```

Comment:
Yikes, bit scary that this was so easy to miss...

Reply:
Yeah :fearful:

Reply:
Maybe we could wrap this up into `OptimadeField` with a custom `Config` that disallows incorrect keys?

Reply:
(If that's even possible)